### PR TITLE
Create riot

### PR DESCRIPTION
**Pull request by EpLiar**

### CODE REVIEW

File: data/riot [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+lol-europe.com
+lolclub.org
+lolespor.com
+lolesports.com
+lolfanart.net
+lolpcs.com
+lolshop.co.kr
+lolstatic.com
+lolusercontent.com
+lpl.cn
```
Comment (Robot-DaneelOlivaw) [inline data/riot:48]: This domain is for sale.

File: data/geolocation-!cn [hunk 2]
```diff
@@ -71,6 +71,7 @@ include:mozilla
 include:nvidia
 include:oracle
 include:razer
+include:riot
```
Comment (Robot-DaneelOlivaw) [inline data/geolocation-!cn:74]: It's more suitable to include under `# Entertainment & Videos & Games & Music & Podcasts`.

Comment (EpLiar) [reply data/geolocation-!cn:74]: OK.

I'm in busy now, please mark this pr as draft.

Comment (EpLiar) [reply data/geolocation-!cn:74]: Only lpl.cn is for sale, right? You show L45-L48 in comment.

Comment (EpLiar) [reply data/geolocation-!cn]: > This domain is for sale.

Removed.

> It's more suitable to include under `# Entertainment & Videos & Games & Music & Podcasts`.

Fixed.

Comment (Robot-DaneelOlivaw) [reply data/geolocation-!cn]: > Only lpl.cn is for sale, right?

Yes. Thanks for your fix.

> You show L45-L48 in comment.

Did I? That's a mistake.

**Pull request merged**